cluster/emulation.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: allow-emulation
```

Comment:
@stu-gott why is this one called `allow-emulation`, but in the documentation `virt-controller`? Can't you just name it `kubevirt-config`?

Reply:
Good catch~
This manifest was added by mistake. I'm not sure you can create a ConfigMap from a manifest like this anyways.